File: Célia_LingProgram/ListaJavaFacul/src/L3_Exer09_Potencia/Exer09_Potencia.java
package L3_Exer09_Potencia;

import java.util.Scanner;

public class Exer09_Potencia {
    public static void main(String[] args) {
        Integer base, expoente, contador = 1, a;


        Scanner leitor = new Scanner(System.in);
        System.out.println("---Calcule a Potência---");
        System.out.println("Insira a base:");
        base = leitor.nextInt();
        System.out.println("Insira o expoente:");
        expoente = leitor.nextInt();
        a = base;

        while (contador < expoente) {
            a *= base;
            System.out.println(contador + ") resulRod: " + a);
            contador++;
        }
        System.out.println(base + " elevado a " + expoente + " = " + a);
    }
}

File: Célia_LingProgram/ListaJavaFacul /src/L3_Exer09_Potencia/Exer09_Potencia.kt
package Lista03

import java.util.*

//
//Elaborar um programa em Java que apresente como resultado o valor de uma potência de uma base qualquer elevada a um
// expoente qualquer, ou seja, de BE, em que B é o valor da base e E o valor do expoente. Considere apenas a entrada de
// valores inteiros e positivos. Por exemplo: B = 2 e E = 5. O programa deverá calcular e exibir o valor de 2 elevado a
// 5 = 32. (Não é para usar métodos que calculam a potência diretamente).

fun main() {
    val scanner = Scanner(System.`in`)
    println("Calcule a potencia")
    println("Insira sua base:")
    val base = scanner.nextInt()
    println("Insira seu expoente:")
    val expoente = scanner.nextInt()

    var a= base
    var contador = 1

    while(contador < expoente){
        a *= base
        println("$contador) ResulRod: " + a)
        contador++
    }

    println( "$base elevado a $expoente = $a")

}